google/cloud/apigee/registry/applications/v1alpha1/diff_analytics.proto

```diff
@@ -43,3 +43,27 @@ message ClassifiedChanges {
   Diff unknownChanges = 3;
 }
 
+/* ChangeStats holds information relating to a list of diffs*/
+message ChangeStats {
+  // total_changes repersents the total number of changes.
+  int64 total_changes = 1;
+  // total_breaking_changes repersents the total number of breaking changes.
+  int64 total_breaking_changes = 2;
+  // total_non_breaking_changes repersents the total number of non-breaking changes.
+  int64 total_non_breaking_changes = 3;
+  // num_diffs repersents the number of diffs used in this stats
+  int64 num_diffs = 4;
+  }
+
+/* ChangeMetrics holds metrics about a list of diffs. Each metric is computed from
+two or more stats. */
+message ChangeMetrics {
+  ChangeStats stats = 1;
```

Comment:
I don't think we need to attach the input message as part of the output message now that we're computing metrics from the stats.

Reply:
Done